### DISCUSSION TITLE
Please tell us why?

### DISCUSSION THREAD

**Discussion by pi0** [pi0/warning.wtf#1] [Announcements]
You are here because you probably saw too many warnings already. 

Well, that's not good. also, it is not always your fault but we love to hear more about your warnings and why you chose to ignore them! So please share your stories!

**Comment by saifobeidat** [comment]
- Duplicated warnings. -- some warnings are the same, which makes our devtools log full of duplicated ones, it would be easier if you can group them (accordion style), and add the count beside the warning group

- They all have the same theme (same background color and text size/color) --  maybe if we can add a bold label of the warning source?
- Sometimes, we ignore them because we can't fix them, they caused by another lib.. ex: in one of the recent vue releases, it started to warn us on a certain functionality that caused Nuxti18n library -- that made our log full of  warnings, which made it hard to trace other warnings.. maybe here if can have an option to hide it temporarily?... [more details    ](https://github.com/vuejs/core/discussions/9974)

**Comment by webpro** [comment]
Nice initiative @pi0, it's a real concern.

Guess I'm here to share an approach I would **not** recommend (still good to know about). Here's how [Rush](https://rushjs.io) tries to "fix" this: it comes down to "promote" warnings to errors so they can't be ignored. It's a default setting that can be changed, but I think this is so fundamentally wrong and pedantic I had to rant about it twice:

- https://github.com/kadena-community/kadena.js/pull/442#issuecomment-1604214895
- https://github.com/microsoft/rushstack/issues/1329#issuecomment-1728152912

My apologies if it's in the wrong discussion, I'd be happy to (re)move it.

Also here to learn about better approaches and solutions :)
